app/seeders/dinosaur_seeder.rb

```diff
@@ -0,0 +1,28 @@
+class DinosaurSeeder
+  DINOSAURS = [
+    {
+        name: "Giganotosaurus",
+        location_found: "South America",
+        info_url: "https://en.wikipedia.org/wiki/Giganotosaurus",
+        user: User.first,
+        description: "This dinosaur is giganotic!"
+      }, {
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.